assets/src/edit-story/components/inspector/document/documentInspector.js

```diff
@@ -23,19 +23,19 @@ import { __ } from '@wordpress/i18n';
 /**
  * External dependencies
  */
-import styled, { css } from 'styled-components';
+import styled from 'styled-components';
 
 /**
  * Internal dependencies
  */
 import { useStory } from '../../../app/story';
 import { useConfig } from '../../../app/config';
 import { SimplePanel } from '../../panels/panel';
-import UploadButton from '../../uploadButton';
+import { useMediaPicker } from '../../mediaPicker';
 import { SelectMenu, InputGroup } from '../../form';
 import useInspector from '../useInspector';
 
-const buttonStyles = css`
+const Button = styled.div`
```

Comment:
Button is a `div`? If that's a critical part, let's add a comment explaining why or reflect this in the name. But maybe there's a way to make it a `button`?